core/src/test/kotlin/de/fraunhofer/iem/spha/core/strategy/WeightedMinimumKPICalculationStrategyTest.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2024 Fraunhofer IEM. All rights reserved.
+ *
+ * Licensed under the MIT license. See LICENSE file in the project root for details.
+ *
+ * SPDX-License-Identifier: MIT
+ * License-Filename: LICENSE
+ */
+
+package de.fraunhofer.iem.spha.core.strategy
+
+import de.fraunhofer.iem.spha.core.hierarchy.KpiHierarchyNode
+import de.fraunhofer.iem.spha.model.kpi.KpiId
+import de.fraunhofer.iem.spha.model.kpi.KpiStrategyId
+import de.fraunhofer.iem.spha.model.kpi.RawValueKpi
+import de.fraunhofer.iem.spha.model.kpi.hierarchy.KpiCalculationResult
+import de.fraunhofer.iem.spha.model.kpi.hierarchy.KpiEdge
+import de.fraunhofer.iem.spha.model.kpi.hierarchy.KpiNode
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class WeightedWeightedMinimumKPICalculationStrategyTest {
```

Comment:
Is this 2x `Weighted` duplicate?
```suggestion
class WeightedMinimumKPICalculationStrategyTest {
```
If yes, the[ WeightedWeightedMaximumKPICalculationStrategyTest](https://github.com/fraunhofer-iem/software-product-health-analyzer/blob/02071d3115f14b0fd2b9fc237896536afdab0df5/core/src/test/kotlin/de/fraunhofer/iem/spha/core/strategy/WeightedMaximumKPICalculationStrategyTest.kt#L22) would also needs to change.